pdns/dnsdist-lua-actions.cc

```diff
@@ -127,14 +127,15 @@ class TeeAction : public DNSAction
 {
 public:
   // this action does not stop the processing
-  TeeAction(const ComboAddress& ca, bool addECS=false);
+  TeeAction(const ComboAddress& rca, const ComboAddress& lca, bool setLocalBindAddress=false, bool addECS=false);
   ~TeeAction() override;
   DNSAction::Action operator()(DNSQuestion* dq, std::string* ruleresult) const override;
   std::string toString() const override;
   std::map<std::string, double> getStats() const override;
 
 private:
   ComboAddress d_remote;
+  ComboAddress d_local;
```

Comment:
Since we only use the local address in the constructor, I don't think we need to keep it around?

Reply:
Possibly using that `d_local` in statistical function?

Reply:
I would prefer not keeping it for now, we can always add it back later when we actually decide to do something with it :)

Reply:
OK, I will make a commit

Reply:
Commit in https://github.com/PowerDNS/pdns/pull/11889/commits/f7bf9ae833f4de1be3a6fe9374f13880678e9ffb